src/pages/favorites/favorites.component.spec.ts

```diff
@@ -71,4 +71,37 @@ describe('Favorites Component', () => {
 
     });
   });
+  describe('removeRoute', () => {
+    // Make an arbitrary route object available to each test
+    beforeAll(() => {
+      this.route = {
+        RouteId: 20035,
+        GoogleDescription: 'Campus Shuttle Southbound',
+        ShortName: '35',
+        RouteAbbreviation: '35',
+        Color: '00467E'
```

Comment:
This is indeed a favorite route - in this case specifically, I threw a debugger into the FavoritesComponent and copied one of _my_ favorite routes when it was loaded onto the page.

I don't know anything about factories here, but I hope so!  For this test, we technically could use any object with the `RouteId` property set, as that's all the code we're testing is looking for.